lib/apiservers/engine/backends/volume.go

```diff
@@ -234,7 +242,7 @@ func translateInputsToPortlayerRequestModel(name, driverName string, opts, label
 	model.Metadata = make(map[string]string)
 	model.Metadata[dockerMetadataModelKey] = metadata
 	if err := validateDriverArgs(opts, model); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Bad Driver Arg: %s", err)
```

Comment:
Should we potentially clean things up? right now an invalid capacity will yield the message `Bad driver arg: Invalid capacity: HelloWorld` if you passed in `--opt Capactiy=HelloWorld` on the create call.